### ISSUE TITLE
USER-COLVARS output

### ISSUE THREAD

**Issue by stanmoore1** [lammps/lammps#752]
@giacomofiorin our internal regression tests are failing for some of the colvars examples due to output from the colvars library. The output from colvars is breaking the thermo output parser because it is intermixed with the thermo output. An example of the output is `colvars: Saving collective variables state to "out2.colvars.state"`. Is there any way to reduce output from colvars?

```
Step Temp TotEng PotEng KinEng E_pair E_bond f_2 
     200    251.50475   -5557.4251   -6569.2538    1011.8288   -6674.0845    24.804906    7.1387572 
     210    253.15303   -5538.5615   -6557.0214    1018.4599   -6672.0497    37.676621   0.61219488 
     220    245.19621   -5522.5191   -6508.9679    986.44887   -6628.1899    36.657688  0.048643392 
     230    258.69885   -5495.7275   -6536.4989    1040.7713   -6658.2886    34.857911   0.22092559 
     240    260.79633   -5469.8678   -6519.0775    1049.2097     -6624.18    31.576952    3.7574815 
     250    269.07527   -5438.3946   -6520.9113    1082.5167   -6616.4382    25.447675    8.6600023 
     260    266.01046   -5397.3486   -6467.5353    1070.1867   -6580.2896    26.871919    8.3323103 
     270    272.81307   -5350.8819   -6448.4362    1097.5543   -6563.8228    23.114196    10.973132 
     280    279.42264   -5307.9798    -6432.125    1124.1452   -6557.3367    33.644022    8.5490488 
     290    286.85168   -5260.8409   -6414.8739     1154.033   -6515.6796    28.574838    5.9100102 
     300    291.52795   -5216.2879    -6389.134    1172.8461   -6503.1275    27.889149    2.2482427 
colvars: Saving collective variables state to "out2.colvars.state".
Loop time of 1.25524 on 8 procs for 100 steps with 2004 atoms
```

**Comment by stanmoore1** [comment]
To be more clear, our parser is expecting columns of thermo data between "Step" and "Loop time" and if colvars output is between those keywords, the parser fails.

**Comment by akohlmey** [comment]
@stanmoore1 you could just let your parser ignore lines not starting with a number and not starting with "Step" or "Loop".

**Comment by giacomofiorin** [comment]
@stanmoore1 I presume this is the `peptide.colvars` example, which gives me an even more complicated output, coming from SHAKE:
```
     290    286.85168   -5260.8408   -6414.8738     1154.033   -6515.6795    28.574837    5.9100101 
SHAKE stats (type/ave/delta) on step 300
  4 1.111 1.79793e-05 36
  6 0.997005 1.02512e-05 12
  8 1.08 1.85103e-05 18
  10 1.111 9.9884e-06 24
  12 1.08 8.84114e-06 18
  14 0.960008 0 2
  18 0.957203 1.8445e-05 3840
  31 104.52 0.00168383
     300    291.52794    -5216.288    -6389.134     1172.846   -6503.1275     27.88915    2.2482425 
colvars: Saving collective variables state to "out.colvars.state".
```
So the issue may be actually broader than just Colvars.  For the sake of easier parsing, you can safely omit parsing any line prefixed by `colvars`, which eliminates all output from Colvars altogether.

**Comment by stanmoore1** [comment]
I made the parser reject any non-numeric lines between the keywords. That seems to make it more robust, testing now.

**Comment by stanmoore1** [comment]
The change to the parser has fixed the issue, closing #752.
